lib/python/picongpu/plugins/jupyter_widgets/base_visualizer.py

```diff
@@ -0,0 +1,367 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sebastian Starke
+License: GPLv3+
+"""
+
+from .utils import get_all_scans, get_all_sims
+from ...utils.find_time import FindTime
+
+import matplotlib.pyplot as plt
+import os
+from ipywidgets import widgets
+from datetime import datetime
+
+
+class Visualizer(widgets.VBox):
```

Comment:
I would like to call this `BaseWidget` to be consistent with the `data` and `plot_mpl` classes:
```suggestion
class BaseWidget(widgets.VBox):
```